community/cypher/cypher-logical-plans-3.4/src/main/scala/org/neo4j/cypher/internal/v3_4/logical/plans/NodeIndexScan.scala

```diff
@@ -22,12 +22,18 @@ package org.neo4j.cypher.internal.v3_4.logical.plans
 import org.neo4j.cypher.internal.frontend.v3_4.ast.{LabelToken, PropertyKeyToken}
 import org.neo4j.cypher.internal.ir.v3_4.{CardinalityEstimation, IdName, PlannerQuery}
 
+/**
+  * This operator does a full scan of an index, producing one row per entry.
+  *
+  * It's much slower than an index seek, since all index entries must be examined, but also much faster than an
```

Comment:
I think these comments should be about comparing equivalent operations. If all entries in an index have the same value, and you do an index seek against that index with that single value, you are performing something equivalent to an `indexScan`. We could write something about it (typically) returning more rows, but I'd refrain from saying anything about speed

Reply:
That was one of the few docs that existed before, so kept it. Now removed it =).